### ISSUE TITLE
"last 2 versions" considered harmfull

### ISSUE THREAD

**Issue by 0x53A** [SAFE-Stack/SAFE-template#152]
see https://jamie.build/last-2-versions

quoting from there:

--------------

## What to do instead

This would be my personal recommendation: 

```js
"browsers": [
  ">0.25%",
  "not ie 11",
  "not op_mini all"
]
```

**Comment by noppa** [comment]
I think lots of people still want to support IE 11.

**Comment by 0x53A** [comment]
I think the main message is that ``last two versions`` is broken, regardless of your needs.

If you need IE11, then you could either use ``>0.25%`` which implicitly includes IE11, or better, ``>0.25%, IE11``, which explicitly includes it.

**Comment by theimowski** [comment]
Closed by #204
